src/foam/parsers/html/lib.js

```diff
@@ -515,6 +515,19 @@
       },
       function isSelfClosing(nodeName) {
         return selfClosingNodeNames[nodeName];
+      },
+      function unescapeString(str) {
+        if ( ! foam.String.isInstance(str) ) return '';
+        var escapeKeys = Object.keys(escapes).map(function(key) {
+          return `&${key};`;
+        });
+
+        var exp = RegExp(`(?=(${escapeKeys.join('|')}))\\1`, 'g');
```

Comment:
Is this RegExp stateful (i.e., can it not be reused)? If it can be reused, then we should store the RegExp, rather than compute it every time.